mina-archive/values.yaml

```diff
@@ -140,7 +140,7 @@ missingBlocksGuardian:
 
 dumpExporter:
   # -- Enabled dump exporter 
-  enabled: true
+  enabled: false
   # -- Suspend the  dump exporter execution
   suspend: false
```

Comment:
We could also enabled both Jobs by default, but suspend them. That could be handy if we decide to run a job manually for any reason.

Reply:
I agree with it.

Reply:
After discussing it, we agree that:
- guardian should be `enabled` by default, and not suspended
- dump exporter should **NOT** be `enabled`, and not suspended